docs/_templates/generate_configvar_docs/common_pdk_vars.md

```diff
@@ -20,19 +20,19 @@ ${"##"} PDK-Level
 These are variables that affect the entire PDK.
 
 
-| Variable Name | Type | Description | Units |
-| - | - | - | - |
+| Variable Name | Type | Description | Units | Deprecated Names |
+| - | - | - | - | - |
 %for var in pdk_variables:
-| `${var.name}`{#${var._get_docs_identifier()}} | ${var.type_repr_md(for_document=True)} | ${var.desc_repr_md()} | ${var.units or ""} |
+| `${var.name}`{#${var._get_docs_identifier()}} | ${var.type_repr_md(for_document=True)} | ${var.desc_repr_md()} | ${var.units or ""} | ${var.get_deprecated_names_md()|join("<br>")} |
```

Comment:
This BR appears to be breaking everything

![Screenshot 2024-10-09 at 13 51 05](https://github.com/user-attachments/assets/c220f321-a3ab-4ccd-8c81-76ee549a43bc)

Reply:
Where is that? The table renders fine in readthedocs.